sw/device/tests/sim_dv/flash_escalation_reset_test.c

```diff
@@ -222,6 +300,12 @@ bool test_main(void) {
 
   if (rst_info == kDifRstmgrResetInfoPor) {
     alert_handler_config();
+    // Make sure we can receive both the watchdog and alert NMIs.
+    CHECK_DIF_OK(dif_rv_core_ibex_enable_nmi(&rv_core_ibex,
+                                             kDifRvCoreIbexNmiSourceAlert));
+    CHECK_DIF_OK(dif_rv_core_ibex_enable_nmi(&rv_core_ibex,
+                                             kDifRvCoreIbexNmiSourceWdog));
```

Comment:
I thought when flash_ctrl gets a fatal error it blocks access, so the NMI would not be run. This means this test may not check that the reset is for the expected reason. We should at least call check_alert_dump in all_escalation_resets_test.c, and perhaps move that function to alert_handler testutils.

This is one of the reasons why I suggested having this test use all_escalation_resets_test.c, and it can also check that the ISRs are not run.

Reply:
Let me handle this with subsequent PR.